### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by xgroleau**
fix ui not working on buzz

### CODE REVIEW

File: src/application-interface/src/ApplicationInterface.cpp [hunk 1]
```diff
@@ -39,8 +39,9 @@ void ApplicationInterface::setSystemConnectionState(ConnectionState state) {
 }
 
 void ApplicationInterface::setSystemDeviceState(DeviceState state) {
-    LockGuard lock(m_mutex);
+    m_mutex.lock();
```
Comment (g-lesssard) [inline src/application-interface/src/ApplicationInterface.cpp:42]: Out of curiosity, why was this changed? Is it an issue with the LockGuard from Propolis or did the member really needed to be accessed earlier than the return of the function?

**Review by g-lesssard — APPROVED** [review]
LGTM, small question concerning mutex

File: src/application-interface/src/ApplicationInterface.cpp (hunk 1 above)
Comment (g-lesssard) [reply src/application-interface/src/ApplicationInterface.cpp:42]: Or before the syscall to setHexDisplay I would guess?

Comment (xgroleau) [reply src/application-interface/src/ApplicationInterface.cpp:42]: Yea it's because setHexDisplay calls it so there would be a deadlock
